apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -6877,6 +6896,28 @@ suite('conversation.js >', function() {
             sinon.assert.calledWith(Compose.on, 'type', onMessageTypeChange);
           });
         });
+
+        test('correctly setups Compose lock for non-email thread', function() {
+          sinon.assert.calledOnce(Compose.setupLock);
+          sinon.assert.calledWith(Compose.setupLock, {
+            forceType: sinon.match.func
+          });
+
+          assert.isNull(Compose.setupLock.lastCall.args[0].forceType());
+        });
+
+        test('correctly setups Compose lock for email thread', function() {
+          Settings.supportEmailRecipient = true;
+          setActiveThread(threadId, ['nobody@mozilla.com']);
+          ConversationView.beforeEnter(transitionArgs);
+
+          sinon.assert.calledTwice(Compose.setupLock);
```

Comment:
Not sure either, I wanted to be sure that ```lastCall``` is the one I need, but yeah more confusion than value, will remove both these asserts from this test